packages/synapse-react-client/src/lib/containers/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DocumentTemplate.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Typography } from '@mui/material'
+import DirectDownloadButton from '../../DirectDownloadButton'
+import { FileHandleAssociation } from '../../../utils/synapseTypes'
+import IconSvg from '../../IconSvg'
+import React from 'react'
+import { useGetFileBatch } from '../../../utils/hooks/SynapseAPI/file/useFiles'
+
+export type DownloadDocumentTemplateProps = {
+  title: React.ReactNode
+  description: React.ReactNode
+  fileHandleAssociation: FileHandleAssociation
+  downloadButtonText: string
+}
+
+export default function DocumentTemplate(props: DownloadDocumentTemplateProps) {
```

Comment:
Generic version of the DUC template section, since it's likely we'll have other templates in the future.